src/components/datatable.component.ts

```diff
@@ -452,6 +453,13 @@ export class DatatableComponent implements OnInit, AfterViewInit, DoCheck {
   @Output() rowContextmenu = new EventEmitter<{ event: MouseEvent, row: any }>(false);
 
   /**
+   * The context menu was invoked on a header cell.
+   *
+   * @memberOf DatatableComponent
+   */
+  @Output() headerContextmenu = new EventEmitter<{ event: MouseEvent, name: string, prop: string }>(false);
```

Comment:
Non-backwards compatible is fine. Does `contextmenu` actually work? I've had issues in the past with event collisions.